static/mantis-node-ids.nix

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
static files should not be committed

Reply:
Removed.

PS: the following files were already tracked in `master`, so were *NOT* removed:

```
static/id_buildfarm.pub
static/txgen-logging-warnonly.yaml
static/txgen-logging.yaml
```